### ISSUE TITLE
Update LDN cohorts LDN9 on Dashboard

### ISSUE THREAD

**Issue by MattCFry** [CodeYourFuture/CYF-Staff-Tickets#580]
### Background

The dashboard team created this functionality some time ago and needs to use it to ensure it is useful. We will start with cohorts ending 2023/34 and address older cohorts later.

### Acceptance criteria

- [x] All LDN applicants on the dashboard who meet the criteria above are assigned to a cohort.

### Owner

@mattcfry

### Stakeholder

@kfklein15

### Approver

@kfklein15

**Comment by MattCFry** [comment]
**Course Start Dates**

- LDN 2 - 15/03/2017
- LDN 3 - 15/10/2017
- LDN 4 - 11/03/2018
- LDN 5 - 02/12/2018
- LDN 6 - 24/11/2019
- LDN 7 - 18/10/2020
- LDN 8 - 25/09/2021
- LDN 9 - 09/07/2022
- LDN 10 - 09/07/2022

**Comment by MattCFry** [comment]
For London 9+10, I will just put London 9 on the dashboard for relevant applicants.

**Comment by MattCFry** [comment]
I have some blockers. I am[ addressing them](https://codeyourfuture.slack.com/archives/C02FX4W0VM4/p1710158280689199) with the Dashboard team.

**Comment by MattCFry** [comment]
Unblocked. After speaking with Esma on the dashboard channel, I figured out the following way to complete this ticket:

- Download LDN applicants as an Excel file on my machine.
- Filter for applicants after LDN 8 start date and before LDN 9 & 10 start date
- Add their cohort on the dashboard as LDN 9

Given the current circumstances, I'll adopt a similar approach for other tickets.

**Comment by MattCFry** [comment]
@kfklein15, if you get a moment, could you please review? Alternatively, is there a more efficient way of reviewing?

**Comment by kfklein15** [comment]
Hi Matt,
I would suggest starting with London 10 - then London 9, 8, etc. 
I would also suggest doing 1 ticket per cohort, so you can prioritse this what all your other priorities and make the tickets small enough to be done 1 per sprint.

**Comment by MattCFry** [comment]
Hi Karen. Thanks for responding.

In the future, I can do individual tickets for cohorts. 100%

For now, I am limited to the most recent cohorts, as I can only select them on the dashboard. I will have to address historic cohorts another time. Esma and the team seem aware of this. 

Separate issue - LDN 9&10 start dates are the same on Global trainees (09/07/2022). I've put all the relevant applicants as LDN 9 as I cannot differentiate.

**Comment by MattCFry** [comment]
LDN10 SDC 14/1/2023
